src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
```

Comment:
remove empty line

Reply:
Error:  /home/runner/work/jv-oop-advanced/jv-oop-advanced/src/main/java/core/basesyntax/Rectangle.java:2: 'CLASS_DEF' should be separated from previous statement. [EmptyLineSeparator]